### DISCUSSION TITLE
Logger not writing to db ?

### DISCUSSION THREAD

**Discussion by WebPirat** [pocketbase/pocketbase#5712] [Q&A] [answered]
Hello,

i tried out this morning the logging feature

```
	app := pocketbase.New()
	app.Logger().Info("Starting app...")
	app.Logger().Debug(
		"Debug message with attributes!",
		"name", "John Doe",
		"id", 123,
	)
```
	
and it doesnt write to the DB is that intendet or did i do something wrong ?

**Comment by ganigeorgiev** [comment, marked as answer]
It is working correctly. The logs are not written in the database because the app is not bootstrapped/initialized yet (this happens on `app.Bootstrap()` or `app.Start()`) and by default fallback to writing to stderr.

**Reply by WebPirat** [reply]
ah now i got it working thank u
